hangman.py

```diff
@@ -0,0 +1,191 @@
+import os
+
+HANGMAN_ASCII_ART = ("""welcome to the game hangman /
+  _    _                                         
+ | |  | |                                        
+ | |__| | __ _ _ __   __ _ _ __ ___   __ _ _ __  
+ |  __  |/ _` | '_ \ / _` | '_ ` _ \ / _` | '_ \ 
+ | |  | | (_| | | | | (_| | | | | | | (_| | | | |
+ |_|  |_|\__,_|_| |_|\__, |_| |_| |_|\__,_|_| |_|
+                      __/ |                      
+                     |___/""")
+
+MAX_TRIES = (6)
+
+HANGMAN_PHOTOS = {"stage 0": "x-------x", "stage 1": """x-------x
```

Comment:
You could probably only save the diff between the stages and than combine them when printing.
Also why not just save them in a list

Reply:
so like we talk i cant rlly save the diff' but i change it to a list